src/services/createUrlQuery.js

```diff
@@ -0,0 +1,20 @@
+export default function creatUrlQuery(
+  caloriesRangeFrom,
+  caloriesRangeTo,
+  query,
+  healthLabels,
+  dishTypes
+) {
+  let result = `https://api.edamam.com/search?q=${query}&app_id=${process.env.REACT_APP_EDAMAM_API_ID}&app_key=${process.env.REACT_APP_EDAMAM_API_KEY}&from=0&to=30`
```

Comment:
that looks complicated! is this from your api service?

Reply:
Yes, this is url for using API